client/src/partials/locations/locations.html

```diff
@@ -9,32 +9,30 @@
 
 <!-- Menu to manage the various locations -->
 <div class="flex-util">
-  <a href="#/locations/village/" id="village" class="btn btn-default btn-sm" style="margin: 0 2px;">
-    <span class="glyphicon glyphicon-home"></span> {{ 'LOCATION.MANAGE_VILLAGE' | translate }}
+  <a ui-sref="locationsVillage" id="village" class="btn btn-default btn-sm" style="margin: 0 2px;" href>
```

Comment:
Is there a reason these `href`s need to remain? I know that for links the mouse interaction behaviour is not the same without an href element but I'm not aware of this for buttons?